inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -1079,37 +1016,33 @@ def _assign_institution_reference_to_affiliations(author_affiliations, already_m
                 already_matched_affiliations_refs[complete_affiliation['value']] = complete_affiliation['record']
 
 
-def normalize_affiliations(obj, eng):
-    matched_affiliations = {}
-    for author in obj.data.get("authors", []):
+def normalize_author_affiliations(obj, eng):
+    search = Search(index="records-hep", using=current_search_client)
```

Comment:
do wee need to create a new search instance and hardcode the index?

Reply:
it was like that before, I can use LiteratureSearch if that works on next

Reply:
It's ok with `LiteratureSearch`, I would keep it